### PR TITLE
#155224941 - Mock API

### PR DESCRIPTION
**Pull request by NaiRobley**
#### What does this PR do?
Remove hard-coded constants and create mock endpoints
#### Description of Task to be completed?
- The URL's in the actions files need refactoring to avoid the hard coding.
- MockAPI to allow front-end devs to continue working without the backend

#### How should this be manually tested?
- Create a .env file based on .env.template.
- Run in development mode by setting NODE_ENV=development in .env
- Test the endpoints on Postman

#### What are the relevant pivotal tracker stories?
PT Story: [#155224941](https://www.pivotaltracker.com/story/show/155224941)
Github Issue: #36

### CODE REVIEW

File: src/actions/incidentAction.jsx [hunk 1]
```diff
@@ -17,7 +13,7 @@ export const loadIncidentsSuccess = incidents => {
 export const loadIncidents = () => {
   return dispatch => {
     return axios
-      .get(url(), {})
+      .get(config.INCIDENTS_URL, {})
```
Comment (kimobrian) [inline src/actions/incidentAction.jsx]: You don't need the empty  `{}` if you are not providing any options.

Comment (NaiRobley) [reply src/actions/incidentAction.jsx]: Noted

**Comment by danielwangai** [comment]
@NaiRobley I see some of @kimobrian 's work on your PR.

**Comment by kimobrian** [comment]
@danielwangai I didn't merge my PR so @NaiRobley is building on it then I will mege this one.

File: server.js [hunk 2]
```diff
@@ -1,25 +1,68 @@
...
+			hot: true,
+			publicPath: webpackConfig.output.publicPath
+		})
+	);
+	app.use(webpackHotMiddleware(compiler));
+    app.use(express.static(path.join(__dirname, '/')));
+    app.use(bodyParser.json());
+    
+    app.get('/api', (req, res, next) => {
+        console.log('Should server mock data');
```
Comment (RayNjeri) [inline server.js]: is this console.log in use??

Comment (NaiRobley) [reply server.js]: That route is not in use, I have removed it

**Pull request merged**